pkg/config/threescale.go

```diff
@@ -118,7 +118,7 @@ func (t *ThreeScale) GetReplicasConfig(inst *integreatlyv1alpha1.RHMI) map[strin
 
 	if resources.RunningInProw(inst) {
 		setDefaultNumberOfReplicas(1, threeScaleComponents)
-	} else if inst.Spec.Type != string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+	} else if inst.Spec.Type != string(integreatlyv1alpha1.InstallationTypeManagedApi) && inst.Spec.Type != string(integreatlyv1alpha1.InstallationTypeMultitenantManagedApi) {
```

Comment:
moving them, may also allow them to be used here. Really would like to avoid the same logic implemented in different ways in different places. Seems like a prime area to cause bugs when we've forgotten these few exceptions next year!

Reply:
Thanks, good idea, I've moved it.